app/helpers/app-version.js

```diff
@@ -9,15 +9,27 @@ const {
 } = config;
 
 export function appVersion(_, hash = {}) {
-  if (hash.hideSha) {
-    return version.match(versionRegExp)[0];
+  let displayVersion = "";
+
+  if (!hash.hideVersion) {
+    displayVersion = version.match(versionRegExp)[0];
+  }
+
+  if (!hash.hideSha) {
+    if (displayVersion) {
+      displayVersion += "+";
+    }
+    displayVersion += version.match(shaRegExp)[0];
   }
 
-  if (hash.hideVersion) {
-    return version.match(shaRegExp)[0];
+  if (!hash.hideDate) {
```

Comment:
I would prefer to keep the current behavior as default and make showing date opt-in. I would suggest that you change the option to `showDate`.